src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MainConsumerImpl.java

```diff
@@ -113,7 +117,12 @@ private void processMessage(String consumerId,
 
         logger.infoContext(messageId, String.format("%s: Consumed Message from Partition: %s, Offset: %s", consumerId, partition, offset), logMap);
         logger.infoContext(messageId, String.format("received data='%s'", data), logMap);
-        messageProcessorService.processMessage(consumerId, data, failedMessageIds);
+
+        boolean isSuccessful = messageProcessorService.processMessage(consumerId, data);
```

Comment:
the javadoc for this method is now out of date.
Please add the param failed message ids and a brief explanation that if the message fails, the id will be appended to the list.

Reply:
Updated the javadoc